### PR TITLE
Added release notes for prefilling supporting process start forms and…

### PR DESCRIPTION
**Pull request by ivo-ritense**
… drop down list of forms

### CODE REVIEW

File: release-notes/major10/10.6.0/valtimo-frontend-libraries.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ The following bugs were fixed:
     or in the API is now used on first load.
   * Date formats specified in list columns of type date are used to correctly display the date value in the desired
     format.
+* **Form dropdown list on object management page not loading all forms**
```
Comment (mritense) [inline release-notes/major10/10.6.0/valtimo-frontend-libraries.md:91]: Forgot white space?

File: release-notes/major10/10.6.0/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -90,6 +90,8 @@ The following features were added:
   A new endpoint has been added for submitting form when the form was initiated using a processlink:
 
   ```POST /api/v1/process-link/{processLinkId}/form/submission```
+* **Prefilling of start forms for supporting processes**
```
Comment (mritense) [inline release-notes/major10/10.6.0/valtimo-backend-libraries.md:93]: Forgot white space?

Comment (ivo-ritense) [reply release-notes/major10/10.6.0/valtimo-backend-libraries.md:94]: fixed

File: release-notes/major10/10.6.0/valtimo-frontend-libraries.md (hunk 1 above)
Comment (ivo-ritense) [reply release-notes/major10/10.6.0/valtimo-frontend-libraries.md:92]: fixed

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**